README.md

```diff
@@ -12,6 +12,9 @@ As new standards and APIs become available `<model-viewer>` will be improved
 to take advantage of them. If possible, fallbacks and polyfills will be
 supported to provide a seamless development experience.
 
+See: [API](#api), [Examples](https://www.webcomponents.org/element/@google/model-viewer/demo/examples/index.html), [Remix on Glitch](https://glitch.com/edit/#!/remix/model-viewer)
```

Comment:
Let's link to the demo directly and I can add a `Remix on Glitch` button on it

Reply:
Added remix button on the Glitch itself: 
![image](https://user-images.githubusercontent.com/240083/48379797-3a224580-e68a-11e8-83df-36bbc3026787.png)

Reply:
:+1: